api/parse_svg_input_constaints.py

```diff
@@ -95,15 +95,16 @@ def translate_polygons_to_SVG(polygons, viewbox_width, viewbox_height, new_filen
 def duplicate_polygon(polygons, pid, quantity):
     """
     Duplicate a particular polygon shape in the polygond data structure to a certain quantity.
-    The caller of this function should ideally already checked the pid to be valid, and quantity is a finite integer>2
+    The caller of this function should ideally already checked the pid to be valid, and quantity is a finite integer>2.
+    When a piece is duplicated, if user wants a even quantity, then half of the pieces are mirrored (ie. left and right pant)
     """
 
     if pid < 0 or pid >= len(polygons):
         print("invalid pid for piece duplication!")
         return
 
     p = polygons[pid]
-    print("polygon to be duplicated: x:{},y:{}", p.x, p.y)
+    mirror = quantity % 2 == 0  # even quantity
```

Comment:
could even do mirror = !(quantity%2) to be more pythonic ;) but it doesn't matter